gems/manageiq-providers-amazon/app/models/manageiq/providers/amazon/cloud_manager/event_catcher/stream.rb

```diff
@@ -209,7 +209,7 @@ def queue_policy(queue_arn, topic_arn)
   "Id": "#{queue_arn}/SQSDefaultPolicy",
   "Statement": [
     {
-      "Sid": "#{Digest::MD5.hexdigest(queue_arn)}",
+      "Sid": "#{Rails.application.config.digest_class.hexdigest(queue_arn)}",
```

Comment:
Oh, Sid is a character from ToyStory, ehrm, its Statement ID, so basically a uniqe ID for this statement. It does not relate at all to the contents